plasmapy/particles/particle_class.py

```diff
@@ -1206,7 +1206,7 @@ def binding_energy(self) -> u.J:
     @property
     def atomic_number(self) -> Integral:
         """
-        Return the number of protons in an element, isotope, or ion.
+        The number of protons in an element, isotope, or ion.
```

Comment:
I'm so happy to see this!  I've been biting my tongue trying not make any comments. 🤣

Reply:
😬